source/SIL.AppBuilder.Portal/src/lib/locales/en-us.json

```diff
@@ -428,7 +438,14 @@
   "admin_settings_buildEngines_disconnected": "Disconnected",
   "admin_settings_buildEngines_status": "Status",
   "admin_settings_buildEngines_lastUpdated": "Last update",
-  "admin_workflowInstances_title": "Workflow Instances",
+  "admin_settings_jobAdministration_title":"Job Administration",
+  "admin_settings_jobAdministration_test_instructions":"Choose how long the test task should take",
+  "admin_settings_jobAdministration_test_seconds":"seconds",
+  "admin_settings_jobAdministration_test_add":"Add Test task",
+  "admin_settings_jobAdministration_list":"View Job Statuses",
```

Comment:
Is `Job Administration` etc. what we want to go with for this? it was BullMQ before, but I thought it would be good to be more generic

Reply:
Doesn't matter to me. Looks good.